modules/cnv-editing-vm-yaml-web.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-edit-vms.adoc
+
+[id="cnv-editing-vm-yaml-web_{context}"]
+
+= Editing the virtual machine YAML configuration
+
+Edit the YAML configuration of a virtual machine directly within the web console.
+
+Not all parameters can be updated. If you edit values that cannot be changed and click *Save*, an error message indicates the parameter that was not able to be updated.
+
+The YAML configuration can be edited while the virtual machine is *Running*, however the changes will only take effect after the virtual machine has been stopped and started again.
+
+[NOTE]
+====
+Navigating away from the YAML screen while editing cancels any changes to the configuration you have made.
+====
+
+.Procedure
+
+. Click *Workloads* -> *Virtual Machine* from the side menu.
+. Select a virtual machine.
+. Click the *YAML* tab to display the editable configuration.
+.. (Optional) You can click *Download* to download the YAML file locally in its current state.
+. Edit the file and click *Save*.
+
+A confirmation message shows that the modification has been successful, including the updated version number for the object.
```

Comment:
s/successful, including/successful and includes

Reply:
Done.